app/views/pages/get-involved.html.erb

```diff
@@ -1,55 +1,183 @@
 <% meta_tag :title, 'Get involved - Teach Computing' %>
-<%= render 'components/hero', hero_title: 'Get involved', subtitle: 'Help us inspire the computing experts of the future', colour: 'lime-green', full_width: true %>
-<div class="govuk-width-container">
-  <div class="govuk-main-wrapper govuk-!-padding-bottom-0">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds ncce-about__content">
-        <p class="govuk-body-l govuk-!-margin-bottom-0">The National Centre for Computing Education is funded by the Department for Education and supporting partners, and delivered by STEM Learning. We provide resources, training and support for teaching computing in schools and colleges from primary through to A level.</p>
+<%= render 'components/hero', hero_title: t(".title"), small_title: true, colour: 'lime-green', full_width: true %>
+<div class="get-involved-page">
+  <div class="govuk-width-container">
+    <div class="govuk-main-wrapper govuk-!-padding-bottom-0">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters ncce-about__content ncce-section">
+          <p class="govuk-body-m"><%= t(".intro.body_1") %></p>
+          <p class="govuk-body-s govuk-!-margin-bottom-0"><%= t(".intro.body_2") %></p>
+        </div>
+
+        <div class="govuk-grid-column-one-quarter ncce-enrichment-card govuk-!-padding-4 ncce-section">
+          <h2 class="govuk-heading-s icon-chart"><%= t(".intro.box.heading") %></h2>
+          <p class="govuk-body-s"><%= t(".intro.box.body") %></p>
+        </div>
```

Comment:
Although the icon chart is referenced in the class description, it isn't rendering for me locally?

<img width="263" alt="Screenshot 2024-06-05 at 15 01 14" src="https://github.com/NCCE/teachcomputing.org/assets/145940509/a9c571c9-d2a4-4283-8185-0f29ec72a86c">

Reply:
@GDTimi - might need to restart your docker containers but if that doesn't work then give me a shout!

Reply:
Needed a server restart on my side, all good 👍